### PR TITLE
docs: add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by paplo75**
Update contributing.md

### Checklist
- [ ] I have linked to any relevant issues.
- [x] I have updated the documentation.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added necessary labels.
- [x] I have done my best to ensure that my PR adheres to [the Fuel Labs Code Review Standards](https://github.com/FuelLabs/rfcs/blob/master/text/code-standards/external-contributors.md).
- [x] I have requested a review from the relevant team or maintainers.

### CODE REVIEW

**Review by Braqzen — COMMENTED** [review]
Brand new account huh, wonder who this is :)

Broadly speaking looks fine. Looks to be potentially duplicated from another repo.
Up to the SDK team but make sure to sign the agreement in the bot message and resolve any CI failures.

**Comment by paplo75** [comment]
> Brand new account huh, wonder who this is :)
> 
> Broadly speaking looks fine. Looks to be potentially duplicated from another repo. Up to the SDK team but make sure to sign the agreement in the bot message and resolve any CI failures.

sir it all about if you accept my contribution to the project. i will sign now. thx

File: docs/src/contributing/contributing.md
Comment (digorithm) [inline docs/src/contributing/contributing.md:1]: Could you rename it to `CONTRIBUTING.md` (all uppercase) to follow the standard for other markdown files in this repo?

File: docs/src/contributing/contributing.md [hunk 1]
```diff
@@ -1,3 +1,66 @@
-# Contributing to `fuels-rs`
+# Contributing To Fuel Rust SDK
+
+Thanks for your interest in contributing to the Fuel Rust SDK!
+
+This document outlines the process for installing dependencies, setting up for development and conventions for contributing.
```
Comment (digorithm) [inline docs/src/contributing/contributing.md:5]:
```suggestion
This document outlines the process for installing dependencies, setting up for development, and conventions for contributing.`
```

File: docs/src/contributing/contributing.md [hunk 2]
```diff
@@ -1,3 +1,66 @@
-# Contributing to `fuels-rs`
+# Contributing To Fuel Rust SDK
```
Comment (digorithm) [inline docs/src/contributing/contributing.md:1]:
```suggestion
# Contributing to the Fuel Rust SDK
```

File: docs/src/contributing/contributing.md [hunk 3]
```diff
@@ -1,3 +1,66 @@
...
+
+Thanks for your interest in contributing to the Fuel Rust SDK!
+
+This document outlines the process for installing dependencies, setting up for development and conventions for contributing.
+
+If you run into any difficulties getting started, you can always ask questions on our [Discourse](https://forum.fuel.network/).
+
+
+# Finding Something to Work On
```
Comment (digorithm) [inline docs/src/contributing/contributing.md:10]:
```suggestion
# Finding something to work on
```

File: docs/src/contributing/contributing.md [hunk 4]
```diff
@@ -1,3 +1,66 @@
...
+
+There are many ways in which you may contribute to the project, some of which involve coding knowledge and some which do not. A few examples include:
```
Comment (digorithm) [inline docs/src/contributing/contributing.md:12]:
```suggestion
You may contribute to the project in many ways, some of which involve coding knowledge and some which do not. A few examples include:
```

File: docs/src/contributing/contributing.md [hunk 5]
```diff
@@ -1,3 +1,66 @@
...
+- Push your changes to a branch in your fork of the repository and submit a pull request.
+  - Make sure mention the issue, which is created at step 1, in the commit message.
+- Your PR will be reviewed and some changes may be requested.
+  - Once you've made changes, your PR must be re-reviewed and approved.
+  - If the PR becomes out of date, you can use GitHub's 'update branch' button.
+  - If there are conflicts, you can merge and resolve them locally. Then push to your PR branch. Any changes to the branch will require a re-review.
+- Our CI system (Github Actions) automatically tests all authorized pull requests.
+- Use Github to merge the PR once approved.
+
+Thanks for your contributions!
```
Comment (digorithm) [inline docs/src/contributing/contributing.md:41]:
```suggestion
- Make sure what you want to contribute is already tracked as an issue.
  - We may discuss the problem and solution in the issue.
- Create a Git branch from where you want to base your work. This is usually master.
- Write code, add test cases, and commit your work.
- Run tests and make sure all tests pass.
- Add the breaking label to your PR if the PR contains any breaking changes.
- Push your changes to a branch in your fork of the repository and submit a pull request.
  - Make sure to mention the issue created in step 1 in the commit message.
- Your PR will be reviewed, and some changes may be requested.
  - Your PR must be re-reviewed and approved once you've made changes.
  - Use GitHub's 'update branch' button if the PR becomes outdated.
  - If there are conflicts, you can merge and resolve them locally. Then push to your PR branch. Any changes to the branch will require a re-review.
- Our CI system (Github Actions) automatically tests all authorized pull requests.
- Use GitHub to merge the PR once approved.

Thanks for your contributions!
```

File: docs/src/contributing/contributing.md [hunk 6]
```diff
@@ -1,3 +1,66 @@
...
+close #123
+```
+
+If the pull request links an issue but does not close it, you can use the keyword `ref` like this:
+
+```sh
+ref #456
+```
+
+Multiple issues should use full syntax for each issue and separate by a comma, like:
```
Comment (digorithm) [inline docs/src/contributing/contributing.md:59]:
```suggestion
Multiple issues should use full syntax for each issue and be separated by a comma, like:
```

**Review by digorithm — DISMISSED** [review]
Thanks for the contribution -- Left a few typos and wording/phrasing suggestions.

**Comment by paplo75** [comment]
> Thanks for the contribution -- Left a few typos and wording/phrasing suggestions.

all done

**Review by Br1ght0ne — APPROVED** [review]

**Review by Br1ght0ne — APPROVED** [review]
Hi @paplo75—thanks for your contribution! I went ahead and updated your PR to fix failing CI checks.

**Review by Salka1988 — APPROVED** [review]

**Pull request merged**